examples/gaia_agent/README.md

```diff
@@ -19,9 +19,9 @@ Additionally, the Gaia agent implements the initial planning step, which produce
 
 ## Quickstart
 Perform all the following steps from the top folder of the repo.
-First, you need to install the dependencies for file converters and make folders:
+First, you need to install the dependencies for file converters:
 ```bash
-pip install -r requirements.converters.txt
```

Comment:
as we discussed, we should make using conda optional here

Reply:
I've updated that section, thanks!